data/labs.yaml

```diff
@@ -215,6 +393,15 @@ labs:
         - Carmela
         - Troncoso
       email: carmela.troncoso@epfl.ch
+    description:
```

Comment:
For uniformity with the other descriptions, I'd also use `>` here.

Reply:
Lazy question: what is the difference in yaml if I put `:`, `:>` or `:|`?

Reply:
https://yaml-multiline.info/  ;-)

Reply:
Nice overview, thanks. The indention example doesn't seem to work, though.